lib/tree_sequence_builder.c

```diff
@@ -655,7 +655,7 @@ tree_sequence_builder_make_pc_node(
     }
     min_parent_time -= PC_ANCESTOR_INCREMENT;
     if (min_parent_time <= mapped_child_time) {
-        ret = TSI_ERR_ASSERTION_FAILURE;
+        ret = TSI_ERR_PC_ANCESTOR_TIME;
```

Comment:
Getting a coverage warning here - I assume it is a hard to reach error state?

Reply:
Yeah, exactly. It's a hard error to provoke, and tsinfer test coverage is in general lower than it should be. This one only showed up in the coverage diff because I changed the name to something more sensible.